server/sonar-server/src/main/java/org/sonar/server/qualityprofile/RegisterQualityProfiles.java

```diff
@@ -141,7 +145,7 @@ private boolean verifyLanguage(String language, List<RulesProfile> profiles) {
     if (profileDto != null) {
       changes.addAll(profileFactory.delete(session, profileDto.getKey(), true));
     }
-    profileFactory.create(session, name);
+    profileFactory.create(session, qProfileWsSupport.getDefaultOrganization(session), name);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "getDefaultOrganization"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)